### ISSUE TITLE
ABAPI.GetEquippedBackpack result is always null

### ISSUE THREAD

**Issue by shudnal** [Vapok/AdventureBackpacks#130]
Possible bug in AdventureBackpacks.API.ABAPI.GetBackPackDefinitionFromComponent method
[line 50](https://github.com/Vapok/AdventureBackpacks/blob/85298d6e65d5c4d03ecd390d5f4dd35677516110/AdventureBackpacks/API/Privates.cs#L50) check of 
```
if (isBackpack)
    return null;
```
results in for example API call `AdventureBackpacks.API.ABAPI.GetEquippedBackpack(Player.m_localPlayer)` to be always null.

**Comment by Vapok** [comment]
Can you describe how you are using the API?

Are you using the ABAPI.DLL reference in your code?

**Comment by shudnal** [comment]
On the screenshot I use the API from AdventureBackpacks mod installed.

In the mod where I needed to use the method I used ILRepack to merge AdventureBackpacksAPI.dll and referenced it in the project. The usual way. But currently I don't need that method so I got rid of it.

I can't see how it's related to question really since it's just inverted logic check for me.
Why should it return null if IsBackpack check is true?
